docs/ape17.rst

```diff
@@ -0,0 +1,537 @@
+install it into a virtual environment before running tests or building
+docs, which means that it will be a good test of whether e.g. you have
+declared the package data correctly.
+
+As a starting point, copy over the ``tox.ini`` file generated in :ref:`Step 0 <step0>`.
+You can always then customize it if needed (although it should work out
+of the box).
+
+Once you have done this you should be able to do the following:
+
+Run tests with minimal dependencies::
+
+   tox -e test
+
+Run tests with astropy LTS and Numpy 1.16::
+
+   tox -e test-astropylts-numpy116
+
+Run tests with all optional dependencies::
+
+   tox -e test-alldeps
+
+Run tests with minimal dependencies and the latest developer version of
+numpy and astropy::
+
+   tox -e test-devdeps
+
+Build the documentation::
+
+   tox -e build_docs
+
+Run code style checks on your code::
+
+   tox -e codestyle
+
+The ``{posargs}`` corresponds to arguments passed to ``tox`` after a
+``--`` separator - for example to make pytest verbose in a ``test``
+environment, you can do::
+
+   tox -e test -- -v
+
+Step 13 - Updating your Continuous Integration
+----------------------------------------------
+
+This step will depend on what continuous integration services you use. Broadly
+speaking, unless there are dependencies you need that can only be installed with
+conda, you should no longer need to use ci-helpers to install these. The
+recommended approach is to use the tox file to set up the different
+configurations you want to use, and to then keep the CI configuration as simple
+as possible.
+
+If you use Travis CI, a good place to start is the ``.travis.yml`` file
+generated in :ref:`Step 0 <step0>`, and you can then see if any previous customizations you had
+made need to be copied over. This file shows how one can configure Travis to use
+tox, optionally using conda via ci-helpers to set up Python on MacOS X and
+Windows.
+
+Step 14 - Update ReadTheDocs configuration
+------------------------------------------
+
+With the set-up described in this migration guide, you should be able to
+simplify the configuration for ReadTheDocs. This can be done via a
+``readthedocs.yml`` or ``.readthedocs.yml`` file (the latter is recommended).
+You should be able to copy over the ``.readthedocs.yml`` file created in :ref:`Step 0 <step0>`.
+With this updated file, you should now be able to remove any pip requirements
+file or conda yml file that were previously used by ``readthedocs.yml``.
+
+Step 15 - Coverage configuration
+--------------------------------
+
+Preivously, astropy-helpers expected the coverage configuration to
+be located in ``my_package/tests/coveragerc``. This is now no longer
+necessary, so you can now define the coverage configuration inside
+the ``setup.cfg`` file, which should help reduce the number of files
+to keep track of. Add the following to the bottom of your ``setup.cfg``::
+
+    [coverage:run]
+    omit =
+        my_package/_{{ cookiecutter._parent_project }}_init*
```

Comment:
Is the migration guide `cookiecutter` generated or is this a typo?

Reply:
Fixed, thanks!